code/modules/mob/living/carbon/xenomorph/Facehuggers.dm

```diff
@@ -76,7 +76,9 @@
 		return
 	addtimer(CALLBACK(src, PROC_REF(check_turf)), 0.2 SECONDS)
 	if(stat == CONSCIOUS && loc) //Make sure we're conscious and not idle or dead.
-		go_idle()
+		var/jumps_before = jumps_left
+		go_idle(FALSE)
+		jumps_left = jumps_before
```

Comment:
If you prefer to do this a different way, have the argument for the proc call be what determines whether jumps_left gets reset back to initial or not.